.env

```diff
@@ -1,10 +1,8 @@
-export PORT=8080
-export HUBOT_AUTH_ADMIN=toto
-export HUBOT_IRC_DEBUG=true
-export HUBOT_IRC_NICK=totobot
-export HUBOT_IRC_ROOMS=#webbot-canonical-web-test
-export HUBOT_IRC_SERVER=irc.canonical.com
-export HUBOT_IRC_PORT=6697
-export HUBOT_IRC_USESSL=true
-export HUBOT_RELEASE_NOTIFICATION_SECRET=secret_notification
-export HUBOT_RELEASE_NOTIFICATION_ROOMS=#webbot-canonical-web-test
\ No newline at end of file
+PORT=8080
+HUBOT_AUTH_ADMIN=toto
+HUBOT_RELEASE_NOTIFICATION_SECRET=secret_notification
+HUBOT_RELEASE_NOTIFICATION_ROOMS=web--design
+MATTERMOST_HOST=chat.canonical.com
+MATTERMOST_GROUP=canonical
+MATTERMOST_USER=webbot
+MATTERMOST_TOKEN=test_token
```

Comment:
```suggestion
MATTERMOST_ACCESS_TOKEN=test_token
```

right?

Reply:
no this is another token that I need to rename